src/kaleidoscope/device/Base.h

```diff
@@ -60,6 +60,7 @@ struct BaseProps {
   typedef kaleidoscope::driver::bootloader::None Bootloader;
   typedef kaleidoscope::driver::storage::BaseProps StorageProps;
   typedef kaleidoscope::driver::storage::None Storage;
+  static constexpr char *short_name = "kaleidoscope";
```

Comment:
To silence warnings about ISO C++ not allowing string literals being converted to `char*`, declare as `static constexpr const ...`.